superset/db_engine_specs/base.py

```diff
@@ -326,7 +326,6 @@ class BaseEngineSpec:  # pylint: disable=too-many-public-methods
     # Does database support join-free timeslot grouping
     time_groupby_inline = False
     limit_method = LimitMethod.FORCE_LIMIT
-    time_secondary_columns = False
```

Comment:
Can you also update `db_engine_specs/README.md` and remove references to this attribute?